tests/recipes/click-test-feedstock/README.md

```diff
@@ -21,7 +21,7 @@ Current build status
 ====================
 
 Linux: [![Circle CI](https://circleci.com/gh/conda-forge/click-test-feedstock.svg?style=shield)](https://circleci.com/gh/conda-forge/click-test-feedstock)
-OSX: [![TravisCI](https://travis-ci.com/conda-forge/click-test-feedstock.svg?branch=master)](https://travis-ci.com/conda-forge/click-test-feedstock)
+OSX: [![TravisCI](https://app.travis-ci.com/conda-forge/click-test-feedstock.svg?branch=master)](https://travis-ci.com/conda-forge/click-test-feedstock)
```

Comment:
I guess the bigger question is whether this should still be listed as the OSX build, but we are likely diving into a different issue of updating the the test feedstocks here (as there are other related things that would need updating as well). Filed a separate issue about this ( https://github.com/conda-forge/conda-smithy/issues/1606 )

Reply:
This is a test recipe. It doesn't look like that on recent rerenders.

Reply:
Agreed. Hopefully the issue spells that out better. Though feel free to correct me there if needed